jobs/delorean/jenkinsfiles/discovery/ga/github/Jenkinsfile

```diff
@@ -55,6 +64,55 @@ node {
         println "[INFO] latestRelease = ${latestRelease}"
     }
 
+    stage('Verify GA Release') {
+        if (templatesDir && registryProjectIDs) {
```

Comment:
Should we use "when" here instead https://github.com/integr8ly/ci-cd/blob/master/jobs/delorean/jenkinsfiles/branch/ga/Jenkinsfile#L79. I can't actually remember why this was preferable, maybe something about an empty stage.

Reply:
In the overview, it shows up as an empty stage if `when` is used instead of looking like it 'ran'
![image](https://user-images.githubusercontent.com/9078522/58572482-ab46d600-8233-11e9-9139-7eb4fd0174cc.png)

Looks like `when` only works for Boolean types though. I can add a flag here for `verifyReleaseIsGA` so we can use this in the when. WDYT?

```
Boolean verifyReleaseIsGA = (templatesDir && registryProjectIDs) ? true : false
...
when(verifyReleaseIsGA) {...}
```